recipes/libgd/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+import glob
+import os
+from conans import ConanFile, tools, CMake
+
+
+class LibgdConan(ConanFile):
+    name = "libgd"
+    license = "https://github.com/libgd/libgd/blob/master/COPYING"
```

Comment:
```suggestion
    license = "https://github.com/libgd/libgd/blob/master/COPYING"
```

We don't use the license path, the license name. I see the copying file is a mix, so could you please ask to the author a license, or even a license list?

Reply:
It seems that there was already an issue to use a named license on libgd, but it hasn't had much progress because they have to get their contributors to agree. I commented on https://github.com/libgd/libgd/issues/618 and subscribed, and will follow up if I hear something.

Reply:
Would "BSD-like" suffice?

Reply:
Fair enough, as they didn't decide yet!